core/deployment/src/main/java/io/quarkus/deployment/Capability.java

```diff
@@ -59,7 +59,8 @@
      * Presence of SmallRye OpenTracing.
      */
     SMALLRYE_OPENTRACING,
-    SPRING_WEB;
+    SPRING_WEB,
+    SMALLRYE_OPERAPI;
```

Comment:
```suggestion
    SMALLRYE_OPENAPI;
```

Reply:
Good catch !! Can't believe I have done that :)